src/krotov/propagators.py

```diff
@@ -266,11 +280,17 @@ def _rhs(t, rho, L_list):
         out = np.zeros(rho.shape[0], dtype=complex)
         L = L_list[0][0]
         L_coeff = L_list[0][1]
-        spmvpy_csr(L.data, L.indices, L.indptr, rho, L_coeff, out)
+        if is_qutip5:
+            out = L_coeff * L @ rho
```

Comment:
`rho` and `out` are numpy arrays that need to be wrapped.
```suggestion
                rho = qutip.data.Dense(rho, copy=False)
                out = qutip.data.matmul(L, rho, scale=L_coeff)
```
With `return out.as_ndarray()` at the end.